test/README.md

```diff
@@ -12,7 +12,8 @@ There are two test suites associated with Mapbox GL JS
 
  To run individual tests:
 
- - Unit tests: `yarn test-unit path/to/file.test.js` where the path begins within the `/test/unit/` directory
+ - Unit tests: `yarn test-unit path/to/file.test.js` where path *does not include* `test/unit/` 
+   - e.g. `yarn test-unit ui/handler/scroll_zoom.test.js`
```

Comment:
I found this instruction confusing, took me a while to realize that I needed to *not* include /test/unit/ in the test path to be able to run a test.